package.json

```diff
@@ -58,7 +60,9 @@
     "eslint": "^8.15.0",
     "eslint-plugin-react": "^7.29.4",
     "fhir": "^4.11.1",
+    "gh-pages": "^4.0.0",
```

Comment:
Hopefully this is stable, I once had issues with this library when I updated it.